### ISSUE TITLE
宣传一下Chrome插件

### ISSUE THREAD

**Issue by yatyricky** [sym233/core-values-encoder#7]
[点击安装](https://chrome.google.com/webstore/detail/%E6%A0%B8%E5%BF%83%E4%BB%B7%E5%80%BC%E8%A7%82%E7%BC%96%E7%A0%81%E5%99%A8/egjhgogmjjdjkapjljgmnghigdbaofmi)

仓库：
https://github.com/yatyricky/core-values-encoder-crx

**Comment by yatyricky** [comment]
不知为何，我的信用卡不能购买chrome store开发者，$5一个。
就用了比较复杂的手动安装方式，大佬方便的话，可以注册一个挂商店里面

**Comment by 1a57danc3** [comment]
友善爱国公正敬业自由友善自由友善敬业诚信自由自由诚信民主诚信民主敬业爱国友善爱国平等诚信富强友善爱国敬业友善法治诚信自由爱国友善平等自由诚信富强公正友善爱国公正爱国爱国诚信民主法治

**Comment by yatyricky** [comment]
不是。。是我的（友善爱国自由友善平等诚信平等友善自由民主友善爱国法治敬业自由友善自由爱国友善爱国平等爱国友善法治诚信富强民主）不能用，你要是能购买一个账户，我给你打钱都行😁

**Comment by AndsteLyc** [comment]
希望可以加到商店里😁

**Comment by yatyricky** [comment]
google不让我购买开发者账号，有缘人可以搞一个，我出钱都行

**Comment by 233213fedf** [comment]
其实Opera也能用的
![gf](https://user-images.githubusercontent.com/32429926/46259325-9aaa5880-c50a-11e8-876a-311387b7934d.PNG)

**Comment by yatyricky** [comment]
> 希望可以加到商店里😁

已经上线
[点我](https://chrome.google.com/webstore/detail/%E6%A0%B8%E5%BF%83%E4%BB%B7%E5%80%BC%E8%A7%82%E7%BC%96%E7%A0%81%E5%99%A8/egjhgogmjjdjkapjljgmnghigdbaofmi)
感谢 @idealclover
